src/components/boardPage/TaskCard.tsx/TaskCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./TaskCard.scss";
+import { ITask, TaskStatusEnum } from "../../../interfaces/task";
+
+interface TaskCard extends ITask {}
+
+export const TaskCard: React.FC<ITask> = ({
+  assignedQuantity,
+  creatorName,
+  status,
+  taskId,
+  title,
+}) => {
+  const statusComponent = () => {
```

Comment:
Components should always start with a capital letters; else, you could cause problems during rendering.

Reply:
The "statusComponent" is not really a component, it is a function, I did it like that to reduce code on the JSX area since on that function I have a conditional.